server/src/main/java/io/seata/server/lock/LockerFactory.java

```diff
@@ -39,75 +26,25 @@
  */
 public class LockerFactory {
 
-    /**
-     * The constant LOGGER.
-     */
-    protected static final Logger LOGGER = LoggerFactory.getLogger(LockerFactory.class);
-
-    /**
-     * The constant CONFIG.
-     */
-    protected static final Configuration CONFIG = ConfigurationFactory.getInstance();
-
-    /**
-     * The constant locker.
-     */
-    protected static Locker locker = null;
-
-    /**
-     * The constant lockerMap.
-     */
-    protected static Map<String, Locker> lockerMap = new ConcurrentHashMap<>();
-
     /**
      * The constant lockManager.
      */
-    protected static LockManager lockManager;
+    private static volatile LockManager lockManager;
 
     /**
      * Get lock manager.
      *
      * @return the lock manager
      */
-    public static final LockManager getLockManager() {
+    public static LockManager getLockManager() {
```

Comment:
Locker spi can be separated from this pr.
Please take a look at Mission 53. If you are interested, you can claim this mission.
https://www.yuque.com/cgbcac/aoni5y/uw0n5d

Reply:
All right, I claim it

Reply:
Locker spi already separated from this pr.